tools.mk

```diff
@@ -66,12 +66,6 @@ $(PROTOBUF):
 	@echo "Get GoGo Protobuf"
 	@go get github.com/gogo/protobuf/protoc-gen-gogo@v1.3.1
 
-# used to build tm-monitor binaries
```

Comment:
need to remove it above as well, 
line 45 and from line 50, sorry cant comment there, github 😢

Reply:
done